chemie/customprofile/templates/registration/terms_and_conditions.html

```diff
@@ -0,0 +1,104 @@
+<!-- Modal Structure -->
```

Comment:
This file is an okay solution now, but for future reference I think we should look into making a more permanent solution. The way this is now would mean we have to update three separate TACs (this one, wiki one and pdf) every time we make a minor update. Any suggestions on a solution less prone to errors and inconsistencies?